templates/elasticsearch.yml.j2

```diff
@@ -48,6 +48,14 @@ node.ingest: true
 #
 #node.attr.rack: r1
 #
+{% if inventory_hostname in groups['elasticsearch-hot']|default([]) -%}
```

Comment:
Hyphen (-) will trigger warning message about invalid characters like below, try underscore (_) instead
```[WARNING]: Invalid characters were found in group names but not replaced, use -vvvv to see details```

Reply:
This change will affect old variables such as `elasticsearch-master` or `elasticsearch-data` too. Look around files to see if there are more.